x-pack/plugins/cases/public/files/index.ts

```diff
@@ -20,10 +20,13 @@ const buildFileKind = (owner: Owner): FileKindBrowser => {
   };
 };
 
+export const isRegisteredOwner = (ownerToCheck: string): ownerToCheck is Owner =>
+  ownerToCheck in CASES_FILE_KINDS;
```

Comment:
For security reasons, it is better if we use `Object.hasOwn(CASES_FILE_KINDS, ownerToCheck)`

Reply:
Another option would be to compare against the `OWNERS` array directly: https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/common/constants/owners.ts#L18

I also need something similar:

```

const isValidOwner = (ownerToValidate: string): ownerToValidate is Owner => {
  const foundOwner = OWNERS.find((validOwner) => validOwner === ownerToValidate);

  return foundOwner !== undefined;
};

```

That doesn't guarantee that it is registered though, so if you'd prefer to keep what you have that's fine 👍